src/components/SQForm/useForm.tsx

```diff
@@ -1,11 +1,45 @@
 import React from 'react';
-import {getIn, useField} from 'formik';
+import {
+  FieldHelperProps,
+  FieldInputProps,
+  FieldMetaProps,
+  getIn,
+  useField
+} from 'formik';
 import WarningIcon from '@material-ui/icons/NewReleases';
 import VerifiedIcon from '@material-ui/icons/VerifiedUser';
 
+interface useFormParam {
```

Comment:
The generally accepted format for type and interface names is PascalCase, it makes it easier to distinguish them from regular variables or functions